lib/screens/play/resource_card.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+
+import 'package:treasure_of_the_high_seas/model/resource.dart';
+
+class ResourceCard extends StatelessWidget {
+  final Resource resource;
+
+  ResourceCard(this.resource);
+
+  String _getResourceText() {
```

Comment:
Not helped by the fact that Dart doesn't have proper non-nullable types, which is why you have to specify a default case even after exhausting the enum values :grimacing: 

Thinking it might be less code to have a static map somewhere of `Resource` -> `ResourceProperties`, which could hold the colour, text and so on?